src/DB/providers/db.provider.ts

```diff
@@ -0,0 +1,21 @@
+import { DataSource } from 'typeorm';
+
+export const dbProviders = [
```

Comment:
dodaj do tego typ jaki konkretnie to jest

Reply:
zrobione przez Pawła